bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -9,317 +9,230 @@ metadata:
           "kind": "Search",
           "metadata": {
             "name": "search-v2-operator"
-            "namespace": "open-cluster-management"
           },
           "spec": null
         }
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.15.0
+    createdAt: "2024-05-07T20:20:00Z"
+    operators.operatorframework.io/builder: operator-sdk-v1.34.1
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
   name: search-v2-operator.v0.0.1
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-      - description: Search is the Schema for the searches API
-        displayName: Search
-        kind: Search
-        name: searches.search.open-cluster-management.io
-        version: v1alpha1
+    - description: Search is the schema for the searches API.
+      displayName: Search
+      kind: Search
+      name: searches.search.open-cluster-management.io
+      version: v1alpha1
   description: Installs search components and manages lifecycle
   displayName: search-operator
   icon:
-    - base64data: ""
-      mediatype: ""
+  - base64data: ""
+    mediatype: ""
   install:
     spec:
       clusterPermissions:
-        - rules:
-            - apiGroups:
-                - monitoring.coreos.com
-              resources:
-                - servicemonitors
```

Comment:
If we don't add these permissions to role.yaml, we will run into errors while trying to create servicemonitors, subjectaccessreviews and such. Will have to reconcile both files.

Reply:
So we were adding these manually? I think the correct way would be to add those here. https://github.com/stolostron/search-v2-operator/blob/main/controllers/search_controller.go#L60-L68

Then those should get propagated by `make manifests` and `make bundle`.

Reply:
resolved